doc/source/management/data_at_rest_encryption.rst

```diff
@@ -255,6 +256,23 @@ System Variables
 
 The option turns on encryption of temporary files created by |Percona Server|.
 
+Key Rotation and Scrubbing
```

Comment:
The title mentions "and scrubbing", but I don't see anyhting about scrubbing here? Also, some part of scrubbing are independent of key rotation.

Reply:
Text restructured with Scrubbing being a separate section